src/trace/main.go

```diff
@@ -69,6 +74,9 @@ type vmContext struct {
 }
 
 func (*vmContext) NewPluginContext(contextID uint32) types.PluginContext {
+	if err := setEndianness(); err != nil {
+		proxywasm.LogErrorf("Failed to set endianness: %v", err)
```

Comment:
can we continue in this case? maybe better to set a default endianness and avoid crashing later when using the `nativeEndian`?